modules/bases2fastq/main.nf

```diff
@@ -0,0 +1,45 @@
+process BASES2FASTQ {
+    tag "$meta.id"
+    label 'process_high'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using bases2fastq. Please use docker or singularity containers."
```

Comment:
Would it make sense/are we allowed to add this to conda?

Reply:
We've talked about it and plan to, but the packaging and delivery isn't in my scope, I'm just the end consumer. So if we add it to conda, I'll update it!